### DISCUSSION TITLE
Niederzwehren - Free sample set for GrandOrgue

### DISCUSSION THREAD

**Discussion by klangpost** [GrandOrgue/grandorgue#1356] [General]
Dietrich Wierczeyko from Binatural Pipes has released a new sample set. The organ Niederzwehren is a beautiful organ and will probably be one of the best free sets for GrandOrgue. Just try it out.

Maybe someone can list the page in the wiki?

Youtube Demo: https://www.youtube.com/watch?v=jP4Dp8zR6jw
Download: http://binauralpipes.com/niederzwehren

More organs: http://binauralpipes.com/instrumente

**Comment by jason-barba** [comment]
Most interested. But my console has only two manuals, and this sample set has three. But, it also only has three divisions, Pedal, Hauptwerk and Brustwerk. Are all ranks in Hauptwerk and Brustwerk playable from two manuals?

**Reply by thbeckpr** [reply]
The bottom manual is a coupling manual. You can assign the other two manuals to your console.

**Comment by larspalo** [comment]
I've added the site and its organs to the Sample Sets wiki page.

**Reply by rousseldenis** [reply]
@larspalo thanks. Maybe you can do some advertising for your new sample set that I'm currently testing. It's quite marvelous. I would say it's a great free alternative to Piotr's Erfurt Bussleben one. And I think it has more stops. @larspalo one thing, could you add the respective 'ventiel' buttons in the 'Stops' panel?

**Reply by larspalo** [reply]
I'm glad you like it! It wouldn't be a problem to add the sperrventils to the stop panel, but the question would be why? They are not really essential stops in a modern application and I've taken them out from any GC operations so no confusing things should happen for the user. They are however a part of the original organ so I decided to definitely include them on the main panel(s). I think that a user of the sample set should do what we did at the original organ: just leave them on all the time since the air flow and reservoir was good for it anyway.

**Reply by rousseldenis** [reply]
Yes. My problem with the standard panel is that writings are difficult to read. So, I switched to Stops one as my main one. But I'll set them on at launch and that's solved

**Reply by larspalo** [reply]
Right, I've on purpose set them as on by default and so that GC won't affect their state.

**Comment by midimusic** [comment]
Rather a weird organ - No General Cancel and the stops have absolutely no indication as to which manuals they belong to. The third manual has no stops and appears to be permanently coupled to the other two.
However it does play and sounds reasonable.
Also it runs OK within an 8GB memory.
csw900

**Comment by kerkovits** [comment]
I fell in love with the sound of this organ. I cannot stop playing on it. I feel sorry for my neighbours!

**Comment by klangpost** [comment]
The BEST New Sampleset for GrandOrgue - Niederzwehren Sampleset - Organ Demonstration - PAUL FEY
https://www.youtube.com/watch?v=FbsBaKP9VNs

**Comment by ClarkeBrunt** [comment]
Great to see Paul Fey's demo of this set i.e. http://binauralpipes.com/niederzwehren Since I don't know much about registration, I might learn something from it!

I've found with this set (as with Piotr Grabowski's Friesach organ, see my comments in #1407) that the key action noises are turned off by General Cancel, with no obvious way to turn them back on, other than searching through the 'MIDI Objects' menu and triggering the 'stops' back on again. As @midimusic said, there is no GC on the console, but of course GO provides one in the Combination Setter menu, which can be programmed for midi as usual. And if you 'save' the organ after using GC, then it will start with the noises off and you might not realise that the set has any noises!

I've fixed this by adding 'GCState=-1' in the 6 stops in the ODF ([Stop010] and 011 110 111 307 308).

And if you've only got 2 manuals (like me), then you can ignore the permanently coupled manual (the bottom one), and couple the others as needed using the GO-provided couplers panel.

Later edit: also noticed that the bottom manual (the coupling manual) is de-coupled by GC, similarly with no obvious way to couple it again. So two more places to add 'GCState=-1', making 8 places in all. The wind noise (Windmaschine) already has 'GCState=-1', so isn't turned off by GC.
